db/migrate/20150226010007_add_time_to_event.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimeToEvent < ActiveRecord::Migration
+  def change
+    add_column :events, :time, :time
+  end
+end
```

Comment:
could't roll back because i did a change instead of an up and down, this is not start_time